package-lock.json

```diff
@@ -2082,9 +2082,9 @@
           }
         },
         "node-fetch": {
-          "version": "2.6.5",
```

Comment:
Are you meaning to commit these changes to this file?  I thought that the only time this is updated is when we explicitly want to update the dependencies.